roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Start Nginx
+  service:
```

Comment:
systemd is better here.
also the correct syntax would be:
systemd:
  name: nginx
  state: started